content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -245,6 +241,12 @@ different Kubernetes components.
 | `EnableAggregatedDiscoveryTimeout` | `true` | Deprecated | 1.16 | - |
 | `EnableEquivalenceClassCache` | `false` | Alpha | 1.8 | 1.14 |
 | `EnableEquivalenceClassCache` | - | Deprecated | 1.15 | - |
+| `EndpointSlice` | `false` | Alpha | 1.16 | 1.16 |
+| `EndpointSlice` | `false` | Beta | 1.17 | |
+| `EndpointSlice` | `true` | Beta | 1.18 | |
```

Comment:
👋 @swetharepakula .
Would you fill in the last column for the EndpointSlice `Beta` items and the EndpointSliceNodeName `Alpha` item?

Reply:
I completely missed that. Thanks!